packages/element/README.md

```diff
@@ -25,23 +25,30 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 ## Usage
 
-Let's render a customized greeting into an empty element:
+Let's render a customized greeting into an empty element. 
+
+**Note:** `createRoot` was introduced with React 18, which will be bundled with WordPress 6.2. Therefore it may be necessary to mount your component depending on which version of WordPress (and therefore React) you are currently using. This is possible by checking for an undefined import and falling back to the React 17 method of mounting an app using `render`.
+
+Assuming the following root element is present in the page:
 
 ```html
 <div id="greeting"></div>
-<script>
-	function Greeting( props ) {
-		return wp.element.createElement(
-			'span',
-			null,
-			'Hello ' + props.toWhom + '!'
-		);
-	}
-
-	wp.element
-		.createRoot( document.getElementById( 'greeting' ) )
-		.render( wp.element.createElement( Greeting, { toWhom: 'World' } ) );
-</script>
+```
+
+We can mount our app:
+
+```js
+import { createRoot, render, createElement } from '@wordpress/element';
+
+function Greeting( props ) {
+	return createElement( 'span', null, 'Hello ' + props.toWhom + '!' );
+}
+
+if ( createRoot ) {
+	createRoot( document.getElementById( 'greeting' ) ).render( createElement( Greeting, { toWhom: 'World' } ) );
+} else {
+	render( createElement( Greeting, { toWhom: 'World' } ), document.getElementById( 'greeting' ) );
+}
```

Comment:
In the code sample, better extract the common parts:
```js
const domElement = document.getElementById( 'greeting' );
const uiElement = createElement( Greeting, { toWhom: 'World' } );

if ( createRoot ) {
  createRoot( domElement ).render( uiElement );
} else {
  render( uiElement, domElement );
}
```
This shows the difference between the two APIs much more clearly.

Reply:
I like it... done in fdee779969e8ce4d5765d35feafe4ccee5d34267